libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/VoiceInstructionLoader.java

```diff
@@ -1,202 +0,0 @@
-package com.mapbox.services.android.navigation.v5.navigation;
-
-import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
-
-import com.google.auto.value.AutoValue;
-import com.mapbox.api.directions.v5.models.LegStep;
-import com.mapbox.api.directions.v5.models.VoiceInstructions;
-import com.mapbox.api.speech.v1.MapboxSpeech;
-import com.mapbox.services.android.navigation.v5.routeprogress.RouteProgress;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import okhttp3.Cache;
-import okhttp3.ResponseBody;
-import retrofit2.Call;
-import retrofit2.Callback;
-import retrofit2.Response;
-
-@AutoValue
-public abstract class VoiceInstructionLoader {
```

Comment:
As far as i can see this was only used for the mapbox speed service which i tried to strip away. So this was not used anymore. Do you think we need the Loader in the future?

Reply:
I personally don't mind, as we are not using Mapbox, so that's fine by me, I was mostly wondering, since this seems unrelated to the actual UI change and should probably be a separate PR - since it's already here, I don't mind.